src/appinventor/index.md

```diff
@@ -2,7 +2,12 @@
 title: App Inventor
 ---
 
-Dette er en introduksjon til MIT App Inventor, hvor vi kan lage applikasjoner til Android. 
-� lage apps i MIT App Inventor er en ganske enkel m�te � lage apps p�, men vi anbefaler 
-at en er kjent med programmering i Scratch f�r en pr�ver ut dette. 
-[Sjekk ut MIT App Inventor her.](http://ai2.appinventor.mit.edu/)
\ No newline at end of file
+Her vil vi lære hvordan MIT App Inventor virker, App Inventor er et
+programmeringsmiljø som brukes for å lage apps til Android telefoner
+og nettbrett.
+
```

Comment:
Her vil du lære hvordan MIT App Inventor virker. App Inventor er et programmeringsmiljø som brukes for å lage apps til Android-telefoner og nettbrett.